packages/usa-input-mask/src/content/usa-input-mask-custom-text.json

```diff
@@ -0,0 +1,12 @@
+{
+  "customInvalidAlphaText": "Ingrese un carácter de letra aquí",
+  "customInvalidNumericText": "Ingrese un carácter de número aquí",
```

Comment:
Setting custom invalid text works great and is super easy! I experimented with other templates and everything went smoothly